README.md

```diff
@@ -208,6 +208,42 @@ Which will provide you with:
 {"family":"ubuntu","release":"14.04","arch":null}
 ```
 
+## Supported OS
+
+Remote Targets
+
+Platform | Versions | Architectures
+ ---- | --- | --- | ---
+AIX | 6.1, 7.1, 7.2 | ppc64
+CentOS | 5, 6, 7 | i386, x86_64
+Debian | 7, 8 | i386, x86_64
+FreeBSD | 9, 10 | i386, amd64
+Mac OS X | 10.9, 10.10, 10.11 | x86_64
+Oracle Enterprise Linux | 5, 6, 7 | i386, x86_64
+Red Hat Enterprise Linux | 5, 6, 7 | i386, x86_64
+Solaris | 10, 11 | sparc, x86
+Windows | 7, 8, 8.1, 2008*, 2008R2* , 2012, 2012R2 | x86, x86_64
+Ubuntu Linux | | x86, x86_64
+SUSE Linux Enterprise Server  | 11, 12 | x86_64
+Scientific Linux | 5.x, 6.x and 7.x | i386, x86_64
+Fedora  | | x86_64
+OpenSUSE | 13.1/13.2/42.1 | x86_64
+OmniOS | | x86_64
+Gentoo Linux | | x86_64
+Arch Linux | | x86_64
+
+* Powershell with Windows Management Framework 5.0 required
```

Comment:
Maybe: `For Windows 2008 and 2008 R2 an updated Powershell (Windows Management Framework 5.0) is required.`